src/server/game/AI/SmartScripts/SmartScriptMgr.h

```diff
@@ -1240,10 +1240,16 @@ enum SMARTAI_TARGETS
     SMART_TARGET_FARTHEST                       = 28,   // maxDist, playerOnly, isInLos
     SMART_TARGET_VEHICLE_PASSENGER              = 29,   // TODO: NOT SUPPORTED YET
 
+    SMART_TARGET_TC_END                         = 30,   // placeholder
+
+    // AC-only SmartTargets:
+
+    SMART_TARGET_AC_START                       = 49,   // placeholder
```

Comment:
this is nice! what about making them start to a higher number, to avoid future conflicts with common targets?
for consistency with actions, we could start the AC-only SmartTargets from 200. It would also be easier to remember what actions are supported only by AC when dealing with SAI.

Reply:
Meh, I used those because I have them in every file :-P

I dont think there are more TC Targets to come tho... there have been 5 new ones in 10 years or so xD

If you want me to change it, I'll do it.

Reply:
@P-Kito if you don't mind :) I like to keep consistency and 200+ are safer